airbyte-integrations/connectors/source-mixpanel/unit_tests/test_source.py

```diff
@@ -0,0 +1,48 @@
+from airbyte_cdk import AirbyteLogger
+
+from source_mixpanel.source import SourceMixpanel
+
+logger = AirbyteLogger()
+
+MIXPANEL_BASE_URL = "https://mixpanel.com/api/2.0/"
+
+
+def test_check_connection(requests_mock, config, empty_response_ok):
+    requests_mock.register_uri("GET", MIXPANEL_BASE_URL + "funnels/list", empty_response_ok)
+
+    ok, error = SourceMixpanel().check_connection(logger, config)
+
+    assert ok
+    assert not error
```

Comment:
nit
```suggestion
    assert ok and not error
```

Reply:
changed in [commit](https://github.com/airbytehq/airbyte/pull/11633/commits/569391a5c437021aa9df128581557b16e60a3704)